test/src/test_pylint.py

```diff
@@ -10,13 +10,10 @@
 from .. import test
 
 
-@pytest.mark.skipif(not test.TestBase.have_test_checkout(), "no test-checkout access")
-def test_pylint():
-    #
-    # Run `pylint` on all python sources. We simply use `find` to locate
-    # all `*.py` files, and then manually select the reverse-domain named
-    # modules we have.
-    #
+@pytest.fixture(name="source_files")
+def discover_source_files():
```

Comment:
Technically, we might see source-tests on something other than python files, so maybe `discover_py_sources()?`, but also fine to change in the future, when we actually have a test other than on python sources.

Reply:
Yeah, could make it more explicit.